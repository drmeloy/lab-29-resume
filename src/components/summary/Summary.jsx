import React from 'react';
import PropTypes from 'prop-types';

const Summary = summary => {
  return(
    <>
      <h2>Summary</h2>
      <p>{summary}</p>
    </>
  );
};

Name.PropTypes = {
  summary: PropTypes.string.isRequired
}

export default Summary;
