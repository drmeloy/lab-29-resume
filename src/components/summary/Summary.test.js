import React from 'react';
import { shallow } from 'enzyme';
import Summary from './Summary.jsx';

describe('Summary component', () => {
  it('renders a Summary component', () => {
    const wrapper = shallow(<Summary summary={''} />);
    expect(wrapper).toMatchSnapshot();
  })
});
