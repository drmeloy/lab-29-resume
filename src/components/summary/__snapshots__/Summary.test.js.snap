// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Summary component renders a Summary component 1`] = `
<Fragment>
  <h2>
    Summary
  </h2>
  <p />
</Fragment>
`;
